### ISSUE TITLE
bam.iobio hangs on custom genomes

### ISSUE THREAD

**Issue by nekrut** [galaxyproject/galaxy#3150]
In this history:

https://usegalaxy.org/u/aun1/h/unnamed-history-9

I have mapped reads (datasets 1 and 2) to phiX174 genome (dataset 3). If you click on "visualize in iobio" link than the following happens:

* iobio start up
* it then goes to seemingly infinite loop showing this:

![iobio](https://cloud.githubusercontent.com/assets/4291636/20218396/76dcef80-a7f2-11e6-9b14-7f580c16a612.png)

nothing happens afterwards

I was hoping @blankenberg and @chmille4 can take a look at this

**Comment by chmille4** [comment]
This was a bug on our end and it has been fixed (link below) as best it can. Unfortunately we still can't display the read coverage chart as we use the .bai file to quickly grab read coverage data and there isn't enough information in the .bai file for bam files this small. I've added a warning message to inform the user when this happens.

http://bam.iobio.io/?bam=https%3A%2F%2Fusegalaxy.org%2Fdisplay_application%2Fbbd44e69cb8906b596afe74b20bf08f0%2Fiobio_bam%2Fbam_iobio%2F8337318c7e1aaa32%2Fdata%2Fgalaxy_bbd44e69cb8906b596afe74b20bf08f0.bam&region=all
